metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -197,6 +207,60 @@ def _exec_col_query(self, table_uri: str) -> Tuple:
 
         return sorted(cols, key=lambda item: item.sort_order), last_neo4j_record
 
+    def _get_type_metadata(self, type_metadata_results: List) -> Optional[TypeMetadata]:
```

Comment:
we really want some logging here. High chance we'll end up finding a bug and tracking it down without logging will be annoying.